lib/msal-node/src/retry/LinearRetryStrategy.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import http from "http";
+
+export class LinearRetryStrategy {
+    /**
+     * Calculates the number of milliseconds to sleep based on the `retry-after` HTTP header.
+     *
+     * @param retryHeader - The value of the `retry-after` HTTP header. This can be either a number of seconds
+     *                      or an HTTP date string.
+     * @returns The number of milliseconds to sleep before retrying the request. If the `retry-after` header is not
+     *          present or cannot be parsed, returns 0.
+     */
+    public calculateDelay(
+        retryHeader: http.IncomingHttpHeaders["retry-after"]
```

Comment:
Why do you need this class to depend on http?

Reply:
You're right, I don't need the whole class, but I do need the IncomingHttpHeaders type. I can import just the type from http.

Reply:
I do because this function relies on the IncomingHttpHeaders type, which indicates a number of seconds or an HTTP date string. If I moved this logic outside of the LinearRetryStrategy, then I'd have to duplicate the logic in every retry policy that uses the LinearRetryStrategy.